DirectProgramming/DPC++FPGA/Tutorials/Tools/dynamic_profiler/CMakeLists.txt

```
@@ -1,20 +1,17 @@
 if(UNIX)
     # Direct CMake to use dpcpp rather than the default C++ compiler/linker
     set(CMAKE_CXX_COMPILER dpcpp)
-else() # Windows
-    # Force CMake to use dpcpp rather than the default C++ compiler/linker 
-    # (needed on Windows only)
-    include (CMakeForceCompiler)
-    CMAKE_FORCE_CXX_COMPILER (dpcpp IntelDPCPP)
-    include (Platform/Windows-Clang)
-endif()
-
-cmake_minimum_required (VERSION 3.4)
+    cmake_minimum_required (VERSION 3.4)
 
-project(DynamicProfiler CXX)
+    project(DynamicProfiler CXX)
 
-set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
-set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
+    set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
+    set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
+    set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
 
-add_subdirectory (src)
+    add_subdirectory (src)
+else() # Windows
+    # This code sample does not currently support windows
+    message(STATUS "This code sample is cannot be configured to run on Windows!\
```

Please make this a FATAL_ERROR instead of STATUS message (so CMake will fail).
Typo in the message.
Add 'error' to the message (mostly for our own regtests)
```suggestion
    message(FATAL_ERROR "Error: This code sample cannot be configured to run on Windows!\
```